From 4430142131a367d1c3aecfbe840db4e9afea7739 Mon Sep 17 00:00:00 2001
From: Ajay Kumar Gupta <ajay.gupta@ti.com>
Date: Wed, 23 May 2012 15:32:37 +0530
Subject: [PATCH 448/609] usb: musb: ti81xx: print the usbss revision id
 during init

Original commit: 1957d430ee97a049eec6a22c3f53bb79f5cb0d04

  From git://arago-project.org/git/projects/linux-am33x.git
  And ti-sdk-am335x-evm-05.05.00.00-Linux-x86-Install image

This is to identify the id which can help in debugging.

Signed-off-by: Ravi B <ravibabu@ti.com>
---
 drivers/usb/musb/ti81xx.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/usb/musb/ti81xx.c b/drivers/usb/musb/ti81xx.c
index e3cf38b..f7b2f62 100644
--- a/drivers/usb/musb/ti81xx.c
+++ b/drivers/usb/musb/ti81xx.c
@@ -1069,6 +1069,8 @@ int ti81xx_musb_init(struct musb *musb)
 	if (!rev)
 		return -ENODEV;
 
+	pr_info("MUSB%d controller's USBSS revision = %08x\n", musb->id, rev);
+
 	if (is_host_enabled(musb))
 		setup_timer(&musb->otg_workaround, otg_timer,
 					(unsigned long) musb);
-- 
1.7.5.4

